traceapp/vis.go

```diff
@@ -92,6 +92,11 @@ func (a *App) d3timelineInner(t *appdash.Trace, depth int) ([]timelineItem, erro
 	}
 	for _, e := range events {
 		if e, ok := e.(appdash.TimespanEvent); ok {
+			// Continue to next iteration
+			// if e.Start() or e.End() are empty time values.
+			if e.Start() == (time.Time{}) || e.End() == (time.Time{}) {
+				continue
+			}
```

Comment:
Hi @slimsag,

Just for the record, why is that it would be better to still show the timespan even there's no start or end for it?

I went ahead and try-out the solution you've mentioned above and got the following results and yes frontend web UI breaks:

**Times serialized:**
![screen shot 2016-02-05 at 16 30 03](https://cloud.githubusercontent.com/assets/1000404/12859670/c97cd182-cc25-11e5-939f-b7d7c04f165a.png)

**Browser:**
![screen shot 2016-02-05 at 16 31 17](https://cloud.githubusercontent.com/assets/1000404/12859684/e60896f6-cc25-11e5-9973-f10133c2c8ea.png)


***

Also I've tried out as `start` -> `Time.Now()` and `end` -> `Time.Now().Add(1 * time.Millisecond)` and got the following results, still frontend web UI breaks:

**Times serialized:**
![screen shot 2016-02-05 at 16 32 18](https://cloud.githubusercontent.com/assets/1000404/12859716/1cf49372-cc26-11e5-8dc7-10e8c47d6e01.png)

**Browser:**
![screen shot 2016-02-05 at 16 32 26](https://cloud.githubusercontent.com/assets/1000404/12859725/28fa1c96-cc26-11e5-9d8a-00d2f2dc365d.png)